user/cdtokev.c

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Since we can have long filenames now, you can give this a more descriptive name.